tests/core/actions/test_forms.py

```diff
@@ -761,6 +786,10 @@ async def test_validate_slots_on_activation_with_other_action_after_user_utteran
 
         form_action = FormAction(form_name, action_server)
 
+        mocked.post(
```

Comment:
Why are these lines required now? 🤔

Reply:
Because the validation action now runs during the subsequent lines, it throws a "can't reach action server" exception without these lines.

Reply:
Oh I see, because we have to mimic running the validation action at form initialization 👍🏻